### DISCUSSION TITLE
What is a good way to specify string column and table names?

### DISCUSSION THREAD

**Discussion by cemremengu** [go-jet/jet#185] [Q&A] [answered]
I would like to create statements where user can specify column and table names. What is good way to do this? Something like:

```
SELECT(Node.AllColumns, "schema.user_column").FROM("schema.sometable")
```

**Comment by go-jet** [comment, marked as answer]
Hi @cemremengu,

You can specify single column, the same way as the list of columns:

```golang
SELECT(
    Node.AllColumns,
    Node.SingleColumn,
).FROM(
    Node,
)
```

Check the [wiki](https://github.com/go-jet/jet/wiki).

**Reply by cemremengu** [reply]
Sorry I worded my question wrong. I actually want to specify a column(s) as a string or list of strings.

**Reply by go-jet** [reply]
Aha, I see. You can use column and table constructors:

```golang
SELECT(
    Node.AllColumns,
    StringColumn("sometable.user_column"), // or IntegerColumn, FloatColumn, ...
).FROM(
    NewTable("schema", "sometable", ""),
)
```

**Reply by go-jet** [reply]
Or raw expressions - https://github.com/go-jet/jet/wiki/Expressions#raw-expression

```golang
SELECT(
    Node.AllColumns,
    Raw("sometable.user_column"), // or RawInt, RawString, ....
)
```
